packages/uma-auth-client/src/steps/ConnectedWallet.tsx

```diff
@@ -126,13 +131,7 @@ export const ConnectedWallet = () => {
           connection={connection}
           balance={{
             amountInLowestDenom: balance?.balance || 0,
-            currency: {
-              code: "USD",
-              name: "US Dollar",
-              symbol: "$",
-              decimals: 2,
-              type: "fiat",
-            },
+            currency,
```

Comment:
hmm sorry I think I didnt understand this part earlier :( let me know if im understanding this correctly! but the balance nwc response should have a currency code associated with it, and if its empty I believe we can assume its sats? 
https://github.com/uma-universal-money-address/uma-auth-client-android/pull/24/files#diff-33e2f9c4bb0bdc08a8df0f253d3d780972b58d8cb3cceab9e9e9748fde49a269R59 this may change a bit in pr review but this is how i did it on android